bftengine/src/bftengine/ReplicasInfo.hpp

```diff
@@ -107,6 +107,7 @@ class ReplicasInfo {
   const std::set<PrincipalId> _idsOfExternalClients;
   const std::set<PrincipalId> _idsOfClientServices;
   const std::set<PrincipalId> _idsOfInternalClients;
+  PrincipalId _operator_id;
```

Comment:
Document that there can only be a single operator in the network

Reply:
done